modules/UI/toolbars/Toolbar.js

```diff
@@ -307,45 +360,14 @@ Toolbar = {
         this.toolbarSelector = $("#mainToolbarContainer");
         this.extendedToolbarSelector = $("#extendedToolbar");
 
-        // First hide all disabled buttons in the extended toolbar.
-        // TODO: Make the extended toolbar dynamically created.
-        UIUtil.hideDisabledButtons(defaultToolbarButtons);
```

Comment:
it is duplicate check. We don't need check it here
it does:
`return interfaceConfig.TOOLBAR_BUTTONS.indexOf(name) !== -1
                || interfaceConfig.MAIN_TOOLBAR_BUTTONS.indexOf(name) !== -1;
    },`